src/shared/components/query/QueryStore.ts

```diff
@@ -1450,6 +1450,15 @@ export class QueryStore {
         return new Set(referenceGenomes);
     }
 
+    @computed get readPermissions(): Set<string> {
+        const studies = Array.from(this.treeData.map_node_meta.keys()).filter( s => typeof((s as CancerStudy).readPermission) !== 'undefined' );
+        console.log(studies);
+        const readPermissions = studies
+            .map(n => (!!((n as CancerStudy).readPermission)).toString())
+            .filter(n => !!n);
```

Comment:
Could you add some comments about how to get `readPermissions`? I think it's not very straightforward.

Reply:
comment added in commit